frontend/js/src/about/add-data/AddData.tsx

```diff
@@ -61,6 +61,12 @@ export default function AddData() {
             <code>foo_listenbrainz2</code>
           </a>
         </li>
+        <li>
```

Comment:
Markup looks fine, but please move it down to the "Mobile devices" section (line 363).

I know it's past the page break, but we've been thinking about adding a sort of wizard to help users find the right solution for their use case, in which case if they select iOS, Jewelcase will be the only available option.